compiler/lib/src/main/scala/util/Version.scala

```diff
@@ -3,6 +3,6 @@ package fpp.compiler.util
 /** The compiler version */
 object Version {
 
-  val v = "[unknown version]"
```

Comment:
Please revert this change. This is a glitch in the way the installer works -- if you abort the installation, it can corrupt this file.